build/scripts/testing.mk

```diff
@@ -173,12 +173,15 @@ start-ethereumjs:
 	-p 30303:30303 \
 	-p 8545:8545 \
 	-p 8551:8551 \
-	ethpandaops/ethereumjs:stable \
+	ethpandaops/ethereumjs:master \
```

Comment:
Is there a reason we're using master instead of stable?

Reply:
dialing stable doesn't connect correctly but does with master

Reply:
Yeah arch is right, seems some issue with stable, it is like 6 months old.